configFiles/kubernetes/auctionworkloadcontroller.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wkldcontroller-config
+  labels:
+    app: auction
+    tier: driver
+    type: controller
+data: 
+  PORT: "7500"
+  JVMOPTS: "-Xmx6g -Xms6g -XX:+AlwaysPreTouch"
+  WORKLOADNUM: "1"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wkldcontroller
+  labels:
+    app: auction
+    tier: driver
+    type: controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: auction
+      tier: driver
+      type: controller
+  template:
+    metadata:
+      name: wkldcontroller
+      labels:
+        app: auction
+        tier: driver
+        type: controller
+    spec:
+      containers:
+      - image: hrosenbe/weathervane-auctionworkloaddriver:2.0.0
```

Comment:
Maybe not related to this pull request but,
Should 'hrosenbe' be a replaced with a generic/invalid value in all of the yaml files?
After this release, those images will be constantly changing for whatever next releases.

Reply:
Good point.  I will change this now.